$OpenBSD: patch-setup_py,v 1.2 2017/06/16 14:35:11 sthen Exp $

setuptools breaks at runtime if this is present. test with e.g. certbot

Index: setup.py
--- setup.py.orig
+++ setup.py
@@ -35,7 +35,6 @@ setup(
     packages=find_packages(exclude=['tests', 'docs']),
     platforms=['Any'],
     long_description=read('README.rst'),
-    install_requires=['future'],
     tests_require=['pytest'],
     test_suite='tests',
     classifiers=[
